xbmc/addons/AddonManager.h

```diff
@@ -295,7 +318,10 @@ namespace ADDON
 
     VECADDONS m_updateableAddons;
 
-    bool GetAddonsInternal(const TYPE& type, VECADDONS& addons, bool enabledOnly) const;
+    bool GetAddonsInternal(const TYPE& type,
+                           VECADDONS& addons,
+                           bool enabledOnly,
+                           bool checkIncompatible = false) const;
```

Comment:
See my previous comment on default parameters. 

But as this is a private method I would not insist in removing the default param here. ;-) 

For same reason I do not moan about introduction of a bool param always decreasing the code readability on the calling side.